src/js/containers/LimitOrder/LimitOrderForm.js

```diff
@@ -139,7 +139,7 @@ export default class LimitOrderForm extends React.Component {
   };
 
   getListWarningOrdersComp = () => {
-    if (!this.props.account) return null;
+    if (!this.props.account) return [];
```

Comment:
Why having this change?

Reply:
This is just a quick fix for crashing LO in mobile after hit `buy` or `sell` button in chart layout. Since the order list is supposed to be an array, the default value should be []